models/public/yolo-v3-tf/yolo-v3-tf.md

```diff
@@ -7,7 +7,17 @@ YOLO v3 is a real-time object detection model implemented with Keras\* from this
 ## Conversion
 
 1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `ffede5` commit).
-2. Follow the instructions in README.md file in that repository to get original model and convert it to Keras\* format.
+2. Follow the instructions in README.md file in that repository to get original model (named `yolov3` in repository) and convert it to Keras\* format **OR** follow these steps:
```

Comment:
How can there be alternatives? You could have only done one of these when you converted the model.

Reply:
I'd propose to rewrite that as follows:
```suggestion
2. Use the folllowing commands to get original model (named `yolov3` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/README.md)  file in the official repository):
```

Reply:
It is just some main items from repo's README.

Reply:
> It is just some main items from repo's README.

So why replicate them?

Reply:
Never mind, I didn't see your changes.

Reply:
Wait, I'm mistaken. I thought you removed the expanded text. In that case, the question remains.

Reply:
Because there are several models in the repo, so we just specifying model.